roles/ceph-common/tasks/checks/check_mandatory_vars.yml

```diff
@@ -23,26 +27,36 @@
     ceph_stable_rh_storage and
     not ceph_stable_rh_storage_cdn_install and
     not ceph_stable_rh_storage_iso_install
+  tags:
+    - package-install
 
 - name: make sure journal_size configured
   fail:
     msg: "journal_size must be configured. See http://ceph.com/docs/master/rados/configuration/osd-config-ref/"
-  when: journal_size|int == 0
+  when:
+    journal_size|int == 0 and
+    osd_group_name in group_names
```

Comment:
@leseb The requirements I have are: 1) install packages on mon nodes, 2) install packages on osd nodes, 3) configure mon nodes and then finally 4) configure osds. 

Are you saying that the mon still needs the configuration for OSDs? What's the best approach here if these all need to happen in separate steps?